docs/en/loading/SQL_transaction.md

```diff
@@ -0,0 +1,79 @@
+---
+displayed_sidebar: docs
+---
+
+import Beta from '../_assets/commonMarkdown/_beta.mdx'
+
+# SQL Transaction
+
+<Beta />
+
+This topic describes how to start a simple SQL transaction to commit multiple INSERT statements in a batch.
```

Comment:
Let's avoid "This topic describes ..." because the first few words of the doc appears in the navigation tiles.
```suggestion
Start a simple SQL transaction to commit multiple INSERT statements in a batch.
```